### ISSUE TITLE
Make doesn't work?

### ISSUE THREAD

**Issue by camdenorrb** [P-H-C/phc-winner-argon2#239]
So I ran the `make` command as shown in the CI and documentation, but it appears to not magically build the desired artifacts. What steps am I missing, are there any dependencies I am missing?

#### Error: https://pastebin.com/FG9CTWGh

**Comment by fabianfrz** [comment]
I have just built it on my machine - your problem is that you try to run shell scripts and commands for Linux and BSD on Windows.

**Comment by sneves** [comment]
Yeah. On Windows, I think `make` will only work under a Cygwin or MSYS2 environment. MinGW by itself is unlikely to be sufficient.
